### PR TITLE
doc(readme): community section

### PR DESCRIPTION
**Pull request by sh0rez**
Adds a community section to mention communication channels and the community call.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,12 +51,33 @@
   <a href="https://tanka.dev/tutorial/overview"><strong>Let's kill some YAML together&nbsp;&nbsp;▶</strong></a>
 </p>
 
+Another Grafana Tanka community call is coming up!
+
+- :calendar: Mark the date: 2020-10-06 16:00 UTC
+- :tv: Join the meet in this doc: https://bit.ly/3czxbFz
```
Comment (RichiH) [inline README.md:57]: Not outside shorteners, please. Use hrefs.

File: README.md [hunk 2]
```diff
@@ -51,12 +51,33 @@
...
+
 ## :rocket: Getting started
 
 To get started, [install Tanka](https://tanka.dev/install) first, and then
 [follow the tutorial](https://tanka.dev/tutorial/overview). This should get you
 on track quickly.
 
+## :busts_in_silhouette: Community
+
+The Tanka community is core to the project. Connect to us using the `#tanka`
```
Comment (RichiH) [inline README.md:67]: That sounds cheesy. What about "We're doing this so people can use it! We would love you to join us. You can use our Slack etc pp."

**Review by RichiH — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sh0rez) [reply README.md:57]: This section shouldn't be here in the first place, forgot to remove it

File: README.md [hunk 3]
```diff
@@ -57,6 +57,34 @@ To get started, [install Tanka](https://tanka.dev/install) first, and then
...
+requests. Use one of the above channels so everyone in the community can
+participate.
+
+---
+
+[![Community
+Call](./docs/img/community-call.png)](https://docs.google.com/document/d/1mEsc0GxlnwbWAXzbIP7tBb6T5WgAI66_0gIWJzqB93o/edit)
+
+Grafana Labs hosts a monthly community call for the Tanka project. Notes and the
+meeting URL can be found in a [Google
```
Comment (RichiH) [inline README.md:81]: Spurious line-break, else LGTM

**Review by RichiH — CHANGES_REQUESTED** [review]

**Review by Duologic — APPROVED** [review]

**Pull request merged**